scripts/container-postCreate.sh

```diff
@@ -3,4 +3,6 @@
 # Run pip install
 pip install -r requirements.txt
 
+pre-commit install
```

Comment:
Sorry not quite clear to me when will this script be executed? what does post create mean?

Reply:
After the container is built, this is run - please see the comment in `devcontainer.json`